nodefs/loopback_linux_test.go

```diff
@@ -198,3 +202,119 @@ func TestCopyFileRange(t *testing.T) {
 	}
 
 }
+
+// Wait for a change in /proc/self/mounts
+func waitMount() error {
+	fd, err := syscall.Open("/proc/self/mounts", syscall.O_RDONLY, 0)
+	if err != nil {
+		return err
+	}
+	defer syscall.Close(fd)
+	pollFds := []unix.PollFd{
+		{
+			Fd:     int32(fd),
+			Events: unix.POLLPRI,
+		},
+	}
+	_, err = unix.Poll(pollFds, 1000)
```

Comment:
this is a bit flaky - it will wake up on any change to /proc/self/mounts, not necessarily due to our filesystem (e.g. filesystem from previos test unmounted). We should grep for our filesystem in a loop I think. This could be related somehow to the fact that TestParallelDiropsHang, if run alone, does not hang on my side.

Reply:
True - fixed